rafttest/network.go

```diff
@@ -31,16 +29,7 @@ type iface interface {
 }
 
 type raftNetwork struct {
-	rand         *rand.Rand
-	mu           sync.Mutex
-	disconnected map[uint64]bool
-	dropmap      map[conn]float64
-	delaymap     map[conn]delay
-	recvQueues   map[uint64]chan raftpb.Message
-}
-
-type conn struct {
-	from, to uint64
+	mq *messageQueue
```

Comment:
Could you extract refactors to `rafttest` package to separate PR that we could merge immidietly?

Reply:
I've stripped off refactors to rafttest from this PR.